docs/source/it/_toctree.yml

```diff
@@ -12,3 +12,11 @@
   - local: autoclass_tutorial
     title: Carica istanze pre-allenate con AutoClass
   title: Esercitazione
+- sections:
+  - sections:
+  - sections:
+  - local: Troubleshoot
```

Comment:
I think there is a problem with toctree. I saw the english version of toctree and troubleshooting is in line 92-93 and it's below and there is 3 sections above it